rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -669,12 +669,11 @@ protected BlockValidator buildBlockValidator() {
 
     protected Genesis buildGenesis() {
         RskSystemProperties rskSystemProperties = getRskSystemProperties();
-        BlockchainNetConfig blockchainConfig = rskSystemProperties.getBlockchainConfig();
         return GenesisLoader.loadGenesis(
                 rskSystemProperties.genesisInfo(),
-                blockchainConfig.getCommonConstants().getInitialNonce(),
+                rskSystemProperties.getNetworkConstants().getInitialNonce(),
                 true,
-                blockchainConfig.getConfigForBlock(0).isRskip92()
+                rskSystemProperties.getActivationConfig().isActive(ConsensusRule.RSKIP92, 0L)
```

Comment:
Can we use the same interface as in the other places?

getActivationConfig().forBlock(0).isActive(ConsensusRule.RSKIP92)